app/views/users/invite.html.erb

```diff
@@ -2,20 +2,24 @@
 
 <h1 class="govuk-heading-l"><%= t 'users.invite.title' %></h1>
 
-<%= form_for @form, url: invite_to_team_post_path, class: 'govuk-form-group' do |f| %>
+<%=error_summary_for(@form&.errors, :form) %>
+<%= form_for @form, url: invite_to_team_post_path, class: 'govuk-form-group', as: :form do |f| %>
```

Comment:
Is it correct that you have a `govuk-form-group` surrounding other `govuk-form-group` `div` elements?

Reply:
Seem's the class statement here was redundant so I have removed it.